projects/tabs/src/modules/vertical-tabset/vertical-tabset-adapter.service.ts

```diff
@@ -2,20 +2,16 @@ import {
   ElementRef,
   Injectable,
   Renderer2,
-  RendererFactory2
+  RendererFactory2,
```

Comment:
Is this comma correct? I didn't think the last item in a list of imports needed a comma after it.